pkg/feature/templates/servicemesh/authorino/base/operator-cluster-wide-no-tls.tmpl

```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.authorino.kuadrant.io/v1beta1
+kind: Authorino
+metadata:
+  name: {{ .AuthorinoConfigs.CrName }}
+  namespace: {{ .Auth.Namespace }}
+spec:
+  image: {{ .AuthorinoConfigs.ContainerImage }}
```

Comment:
As previously suggested - let's remove it for now.

Reply:
Removed